internal/pkg/cli/secret_init.go

```diff
@@ -6,8 +6,10 @@ package cli
 import (
 	"errors"
 	"fmt"
+	"os"
 	"sort"
 	"strings"
+	goTemplate "text/template"
```

Comment:
```suggestion
	txttemplate "text/template"
```
https://rakyll.org/style-packages/#renames-should-follow-the-same-rules